spotify_api.csv

```diff
@@ -0,0 +1,2 @@
+creation_time,auth_token,client_id,client_secret
+null,null,#insert client id here before running,#insert client secret here before running
```

Comment:
Great idea! Will def implement this which will go hand in hand with the generating auth token update.

Reply:
Awesome! Since `.env` shouldn't be included in git, feel free to add a `.env.example` file or something similar that displays the structure of `.env` without including any real values for each variable. Then everyone will know what should go in their actual `.env` file.